src/main/java/com/vaadin/exampledata/ChanceBinary.java

```diff
@@ -0,0 +1,23 @@
+package com.vaadin.exampledata;
+
+import java.time.LocalDateTime;
+import java.util.Random;
+
+public class ChanceBinary extends DataType<byte[]> {

```

Comment:
```suggestion
public class RandomFileData extends DataType<byte[]> {
```

This does not use `chance` for anything so using that in the class name is not necessary

Reply:
Renamed